recipes/scikit-hep/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "scikit-hep" %}
+{% set version = "1.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scikit-hep-{{ version }}.tar.gz
+  sha256: 8b8f29b0a2592ec4d9eac7840655b21fe4bd18336c6397ba15ec02b8d1ac681c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - awkward ~=1.0.0
```

Comment:
These pins are intentionally tight and will be regularly updated upstream to follow developments in each of the packages. These dependencies mostly live under the `scikit-hep` organisation that this packages is named after and it mainly serves as a meta package to combine versions which have been tested together.